### DISCUSSION TITLE
Add a {PLAYTIME} tag

### DISCUSSION THREAD

**Discussion by Tikkle** [EssentialsX/Essentials#3994] [Ideas and Feature Suggestions] [answered]
I noticed in the "/whois" command that EssentialsX tracks a player's Playtime. I was wondering if a tag could be added that can be used in custom.txt, motd.txt, info.txt, etc.

**Comment by triagonal** [comment, marked as answer]
This has now been added with #4098. Update to the [latest version of EssentialsX](https://essentialsx.net/downloads.html) to use the `{PLAYTIME}` tag.
